pkg/services/tokenexchanger/setting_access_token_signing.go

```diff
@@ -0,0 +1,46 @@
+package tokenexchanger
+
+import (
+	authlib "github.com/grafana/authlib/authn"
+	sdkhttpclient "github.com/grafana/grafana-plugin-sdk-go/backend/httpclient"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+const (
+	configSection  = "access_token_signing"
+	configKeyURL   = "exchange_url"
+	configKeyToken = "token"
+)
+
+// This service purely exists for centralizing the exchange config reading from grafana.ini
+type Service struct {
+	Exchanger authlib.TokenExchanger
+}
+
+func ProvideAccessTokenSigner(settings setting.Provider, clientProvider sdkhttpclient.Provider) (*Service, error) {
```

Comment:
Don't really think I need anything for myself other than structs/interfaces in authlib, but not sure how else to make the config reading centralized and still dependent on setting.Provider.

Thoughts @kalleep? I looked at the existing ID signer service and it also doesn't depend on `setting` package for reading the ID signing config.

Reply:
You can also config to https://github.com/grafana/grafana/blob/main/pkg/setting/setting.go#L81 and read it from there.

Reply:
Did that, thanks!